_events/2020-06-14-tool-packaging-workshop.md

```diff
@@ -0,0 +1,27 @@
+---
+site: freiburg
+tags:
+- training
+title: "Workshop on Tool Packaging at BCC2020"
+starts: 2020-07-18
+ends: 2020-07-18
+organiser:
+  name: EOSC-Life-WP2
+location: online
+supporters:
+- eosc
```

Comment:
Can you add http://www.biodaten.info as supporter as well. I guess this needs to be added to the framework.
https://github.com/usegalaxy-eu/website/blob/master/_includes/sponsors.html

We can also add bioconda for example.